server/job/schedule.ts

```diff
@@ -82,6 +83,22 @@ export const startJobs = (): void => {
     cancelFn: () => sonarrScanner.cancel(),
   });
 
+  // Checks if media is still available in plex/sonarr/radarr libs
+  scheduledJobs.push({
+    id: 'availability-sync',
+    name: 'Update availability',
+    type: 'process',
+    interval: 'long',
+    job: schedule.scheduleJob(jobs['availability-sync'].schedule, () => {
+      logger.info('Starting scheduled job: Update availability', {
```

Comment:
```suggestion
    name: 'Media Availability Sync',
    type: 'process',
    interval: 'long',
    job: schedule.scheduleJob(jobs['availability-sync'].schedule, () => {
      logger.info('Starting scheduled job: Media Availability Sync', {
```
Try to be consistent with the job name :)